corehq/apps/app_manager/templates/app_manager/partials/case_list.html

```diff
@@ -203,7 +203,9 @@
                 {% trans "Select Parent First" %}
             </label>
             <div class="col-sm-4 checkbox">
-                <input type="checkbox" data-bind="checked: active"/>
+                <label>
```

Comment:
Didn't mean to put this change in this PR, but it's not hurting anything. It just adjusts the spacing for this checkbox.